docs/source/deployment/lambda.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Deploying with AWS Lambda
+description: How to deploy Apollo Server with AWS Lambda
+---
+
+[AWS Lambda](https://aws.amazon.com/lambda/) is a serverless computing platform with a pay-for-use billing model that enables you to run code without worrying about provisioning or managing servers.
+
+In this guide, we'll walk through how to deploy Apollo Server's [AWS Lambda integration](https://github.com/apollo-server-integrations/apollo-server-integration-aws-lambda) to AWS Lambda using the [Serverless framework](https://www.serverless.com/).
+
+## Prerequisites
+
+Make sure you've completed the following before proceeding with this guide:
+
+- [Create an AWS account](https://aws.amazon.com/free/)
+- [Install the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+- [Create an IAM user](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-creds-create)
+- [Configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) with your new IAM user's credentials
+
+> ⚠️ AWS best practices warn against using your AWS account root user keys for any task where it's not required (e.g., don't use these keys to configure the AWS CLI). Instead, [create an IAM user account](https://docs.aws.amazon.com/IAM/latest/UserGuide/getting-started_create-admin-group.html#getting-started_create-admin-group-console) with the [least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) required to deploy your application, and [configure the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) to use that account.
+
+## Setting up your project
+
+For this example, we'll start from scratch to show how all the pieces fit together.
+
+Begin by installing the necessary packages for using Apollo Server and its integration for AWS Lambda:
+
+```shell
+npm install @apollo/server graphql @as-integrations/aws-lambda
```

Comment:
Does this TypeScript example work without installing `typescript` and explicitly building it somehow (eg does `serverless` magically bundle TypeScript for you or something?)